lib/ansible/module_utils/facts/system/chroot.py

```diff
@@ -22,7 +22,18 @@ def is_chroot():
             is_chroot = my_root.st_ino != proc_root.st_ino or my_root.st_dev != proc_root.st_dev
         except Exception:
             # I'm not root or no proc, fallback to checking it is inode #2
-            is_chroot = (my_root.st_ino != 2)
+            fs_root_ino = 2
+
+            if module is not None:
+                # on Btrfs, the first-created subvolume (root) has inode #256
```

Comment:
On my systems, `stat` shows inode 256 for all subvolumes, not only the first created. Moreover, any subvolume may be mounted on `/`, not only the first one.

Reply:
Hrm, not sure what I meant by that, removed. Thanks.